README.md

```diff
@@ -64,9 +64,54 @@ $ cargo test --release
 A basic [example](https://github.com/poanetwork/hbbft/blob/master/examples/README.md) is included to run a network simulation.
 
 ```
+$ cargo run --example simulation --release
+```
+
+![Screenshot](screenshot.png)
+
+|  Heading    | Definition                       
+| ----------- | -------------------------------------------------------------------------- | 
+| Epoch       |  Epoch number. In each epoch, transactions are processed in a batch by simulated nodes (default is 10 nodes) on a network.                                                            | 
+| Min Time   | Timestamp on the _first_ Tx (transaction) processed in a batch of transactions.                                         |  
```

Comment:
Unsure about wording here around min and max time.

Reply:
Timestamp in sec/ms?

Reply:
Yes, it's in (simulated) milliseconds.

The min time is the time until the first  (correct, i.e. not faulty) _node_ outputs the batch. The max time is the time until the last one outputs the batch. The batch is always output in one piece, with all transactions at once.